Topics/Teamwork.md

```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
 ## Team Communication
 During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines. 
 
+An important part of team communication is in communicating tasks, deadlines and progress. Each team member should have good knowledge of the general progress of the project. This will give each team member a deeper understanding of the current state of the project so that they can more adequetly allocate their time. Perhaps even more importantly, the project manager will be able to have a deeper understanding of the progress of the project, and the team member's working habits and they will be able to manage the team more adequetly.
```

Comment:
A sub-header at the start of the section you added would help signify it. Looking at the above header, I'd suggest using `###`

Reply:
Thanks for the suggestion, is there any titles that you think would work best here?

Reply:
Time management might be an appropriate sub-title to use

Reply:
Thanks for the suggestion, that sounds like a good name.